### PR TITLE
Feat: "WatchLater Page"

### LINKED ISSUE #13: Feat : "WatchLater Page"

### PR DESCRIPTION
**Pull request by sadathshariff**
User can add Videos to Watch later, Remove and See all his Watch later videos

### CODE REVIEW

File: src/components/VideoCard/VideoCard.jsx [hunk 1]
```diff
@@ -18,6 +23,8 @@ export const VideoCard = ({ video }) => {
   const { isLoggedIn } = authState;
   const { liked, watchLater } = videoState;
   const isLiked = isVideoLiked(_id, liked);
+  const isWatchlater = isVideoWatchlater(_id, watchLater);
+  console.log(isWatchlater);
   return (
```
Comment (aishwarya-bm) [inline src/components/VideoCard/VideoCard.jsx:28]: Remove console log

File: src/pages/WatchLater/WatchLater.jsx [hunk 2]
```diff
@@ -1,3 +1,53 @@
...
+              <Button name={"Let's Laugh"} btnvariant={"btn-primary"} />
+            </Link>
+          </div>
+        ) : (
+          <div>
+            <div className="clear-all-div">
+              <h3 className="headline-3">WatchLater - {watchLater.length}</h3>
+            </div>
+            <div className="history-videos">
+              {watchLater.map((video) => (
```
Comment (aishwarya-bm) [inline src/pages/WatchLater/WatchLater.jsx:30]: Usse destructuring here

**Review by aishwarya-bm — APPROVED** [review]
Overall looks good, left small suggestions

File: src/components/VideoCard/VideoCard.jsx [hunk 3]
```diff
@@ -18,6 +23,8 @@ export const VideoCard = ({ video }) => {
   const { isLoggedIn } = authState;
   const { liked, watchLater } = videoState;
   const isLiked = isVideoLiked(_id, liked);
+  const isWatchlater = isVideoWatchlater(_id, watchLater);
+  console.log(isWatchlater);
```
Comment (nishasen) [inline src/components/VideoCard/VideoCard.jsx]: remove the console.logs

**Review by nishasen — APPROVED** [review]
Follow the suggestions given by Aishwarya, everything looks good.

**Pull request merged**